app/services/nsm/importers/xml/import_service.rb

```diff
@@ -34,11 +34,16 @@ def import_xml
         end
 
         def build_message
-          I18n.t('nsm.imports.message',
-                 disbursement: I18n.t('nsm.imports.disbursement').pluralize(claim.disbursements.count),
-                 disbursement_count: claim.disbursements.count,
-                 work_item: I18n.t('nsm.imports.work_item').pluralize(claim.work_items.count),
-                 work_item_count: claim.work_items.count)
+          {
+            title: I18n.t('nsm.imports.message_title'),
+            content: message_content
+          }
+        end
+
+        def message_content
+          "#{I18n.t('nsm.imports.work_items_message', count: claim.work_items.count)}
+            #{I18n.t('nsm.imports.disbursements_message', count: claim.disbursements.count)}
+            </p>#{I18n.t('nsm.imports.message_end')}"
```

Comment:
Why is this hanging closing tag here?

Reply:
having a single closing </p> is equivalent to <p></p>, it adds an empty newline basically

Reply:
oh god forgot markdown would format that

Reply:
but also kind of proves my point haha

Reply:
having a single closing "/<p>" is equivalent to "<p></p>", it adds an empty newline basically

Reply:
<img width="555" alt="image" src="https://github.com/user-attachments/assets/e994d40e-0db6-45e4-9a48-4d2d381a1a82" />

Reply:
That's a bug not a feature and produces invalid HTML. Use `<br>` or just a regular `\n`